lib/diagnostic.rb

```diff
@@ -78,12 +94,12 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a HouseCat, and I go 'meow'"
 
 ## Explain why this would be the output, based on the method lookup chain.
 # your response as a comment here
 ##
-
+The self in the say_hello method refers to the class name which is "HouseCat" and the sound looks into housecat to see the noise so it gets replaced with 'meow'
```

Comment:
The answer that we're looking for involves the method lookup chain.  `say_hello` isn't defined on `HouseCat`, but since `HouseCat`'s superclass is `Cat`, Ruby checks for the existence of `say_hello` on `Cat`.  Since `say_hello` doesn't exist on `Cat`, it looks for it in its superclass, `Animal`, where it's defined.